src/test/java/SaucedemoTesting/runner/CartRunner.java

```diff
@@ -0,0 +1,14 @@
+package SaucedemoTesting.runner;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
```

Comment:
de ce ai doua test runner class? intr-un proiect trebuie sa ai doar unul

Reply:
Pentru ca m-am gandit ca celalalt runner avea path-ul pentru feature-ul cu login insa banuiesc ca doar se modifica in linia respectiva : "features = "src/test/resources/FeatureFilesMG/CartDemo.feature"? O sa sterg unul dintre ele.